### PR TITLE
style: minor fixes of colors

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
Minor fixes of styles are done.
buttons of move to wishlist are styled with primary-color.
image
image

### CODE REVIEW

**Comment by sushmita12345** [comment]
Minor fix of styles so there is no breakpoints, you can proceed to merge.

**Review by sushmita12345 — APPROVED** [review]

File: src/components/Cards/Card.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 :root{
   --badge-bg:#dedfe0;
```
Comment (saifudinbadani) [inline src/components/Cards/Card.css:2]: maybe move this to common css file, it will be helpful in another component as well

**Review by saifudinbadani — APPROVED** [review]
Other than the comment, rest of the code  looks fine.

File: src/components/Cards/Card.css (hunk 1 above)
Comment (vaishnavipaliya11) [reply src/components/Cards/Card.css:2]: kept here because cards are used almost everywhere

**Pull request merged**